activiti-cloud-acceptance-tests/pom.xml

```diff
@@ -17,11 +17,12 @@
     <module>activiti-cloud-acceptance-tests-core</module>
   </modules>
   <properties>
-    <serenity.version>2.3.31</serenity.version>
+    <serenity.version>3.6.23</serenity.version>
```

Comment:
I don't get why this version is specified multiple times: ideally versions should be shared on the root pom once and for all (and packages declaration in the dependencyManagement section).
I guess this cleanup should be done independently given the current status, I'm just posting a comment in case I missed something (?)
cc @erdemedeiros